controllers/cluster_controller.go

```diff
@@ -636,3 +640,11 @@ func validateEksaRelease(ctx context.Context, client client.Client, cluster *any
 	}
 	return nil
 }
+
+func validateExtendedK8sVersionSupport(ctx context.Context, client client.Client, cluster *anywherev1.Cluster) error {
+	bundle, err := c.BundlesForCluster(ctx, clientutil.NewKubeClient(client), cluster)
+	if err != nil {
+		return fmt.Errorf("failed to get bundle for cluster: %w", err)
```

Comment:
We generally want to avoid saying error or failed again right?
```suggestion
		return fmt.Errorf("getting bundle for cluster: %w", err)
```